server/core/src/main/java/dev/slimevr/tracking/processor/skeleton/HumanSkeleton.kt

```diff
@@ -1619,45 +1616,70 @@ class HumanSkeleton(
 		var referenceRotation = IDENTITY
 		headTracker?.let {
 			if (bodyParts.isEmpty() || bodyParts.contains(BodyPart.HEAD)) {
-				// Only reset if head needsMounting or is computed but not HMD
-				if (it.needsMounting || (it.isComputed && !it.isHmd)) {
+				// Only reset if head allowMounting or is computed but not HMD
+				if (it.allowMounting || (it.isComputed && !it.isHmd)) {
 					it.resetsHandler.resetMounting(referenceRotation)
 				}
 			}
 			referenceRotation = it.getRotation()
 		}
 
-		// If onlyFeet is true, feet will be forced to be mounting reset in their reset handlers.
-		val onlyFeet = bodyParts.isNotEmpty() && bodyParts.all { it == BodyPart.LEFT_FOOT || it == BodyPart.RIGHT_FOOT }
 		for (tracker in trackersToReset) {
 			// Only reset if tracker needsMounting
-			if (tracker != null && tracker.needsMounting && (bodyParts.isEmpty() || bodyParts.contains(tracker.trackerPosition?.bodyPart))) {
-				tracker.resetsHandler.resetMounting(referenceRotation, onlyFeet)
+			if (tracker != null && tracker.allowMounting && (bodyParts.isEmpty() || bodyParts.contains(tracker.trackerPosition?.bodyPart))) {
+				tracker.resetsHandler.resetMounting(referenceRotation)
 			}
 		}
 		legTweaks.resetBuffer()
 		localizer.reset()
+
+		if (humanPoseManager.server != null) {
+			humanPoseManager.server.configManager.vrConfig.resetsConfig.lastMountingMethod =
+				MountingMethods.AUTOMATIC
+			if (!humanPoseManager.server.trackingChecklistManager.resetMountingCompleted) {
+				humanPoseManager.server.trackingChecklistManager.resetMountingCompleted = bodyParts.any { it ->
```

Comment:
Is `any` the right choice here? We're essentially just checking if one of the body parts is matching, not all.